### PR TITLE
Make filesystem store part of zep 1

### PR DESCRIPTION
**Pull request by jstriebel**
Since the filesystem store is relatively straight-forward and there were no discussions around it, I propose to make it part of ZEP 1, so that ZEP 1 provides a first usable baseline.

I added a section about it's limitations, which is copied from the core spec, and cleaned up some (IMO obsolete) todos.

I'll provide a separate PR for the actual ZEP to mention that the fs store is included there.

### CODE REVIEW

**Comment by jstriebel** [comment]
@jbms Could you have a look on this one? I think this would be straight-forward to merge, just waiting for an approval.

File: docs/extensions/data-types/complex/v1.0.rst
Comment (rabernat) [inline docs/extensions/data-types/complex/v1.0.rst:14]:
```suggestion
Copyright 2019-2023 `Zarr core development team
```

File: docs/extensions/data-types/datetime/v1.0.rst
Comment (rabernat) [inline docs/extensions/data-types/datetime/v1.0.rst:14]:
```suggestion
Copyright 2019-2023 `Zarr core development team
```

File: docs/extensions/data-types/string/v1.0.rst
Comment (rabernat) [inline docs/extensions/data-types/string/v1.0.rst:14]:
```suggestion
Copyright 2022-2023 `Zarr core development team
```

File: docs/extensions/data-types/struct/v1.0.rst
Comment (rabernat) [inline docs/extensions/data-types/struct/v1.0.rst:14]:
```suggestion
Copyright 2022-2023 `Zarr core development team
```

**Review by rabernat — APPROVED** [review]
Looks good to me, but you may want to update the copyright dates.

**Comment by jstriebel** [comment]
@rabernat Thanks! It seems like it's not necessary to update the copyright years, so I'd rather leave them as-is: https://hynek.me/til/copyright-years/

**Pull request merged**
